src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -0,0 +1,673 @@
+
+import {
+  Alert,
+  Button,
+  Popover,
+  Spinner,
+  Text,
+  TextContent,
+  TextList,
+  TextListItem,
+  TextListVariants,
+  TextListItemVariants,
+  TextVariants,
+} from '@patternfly/react-core';
+import { ExclamationTriangleIcon, HelpIcon } from '@patternfly/react-icons';
+
+import ReleaseLifecycle from './../ImageOutput/ReleaseLifecycle';
+import ActivationKeyInformation from './../Registration/ActivationKeyInformation';
+import { AwsAccountId } from './../TargetEnvironment/Aws/AwsAccountId';
+import {
+  FSReviewTable,
+  PackagesTable,
+  RepositoriesTable,
+} from './ReviewStepTables';
+
+import { RELEASES, UNIT_GIB } from '../../../../constants';
+import { extractProvisioningList } from '../../../../store/helpers';
+import { useAppSelector } from '../../../../store/hooks';
+import { useGetOscapCustomizationsQuery } from '../../../../store/imageBuilderApi';
+import {
+  useGetSourceListQuery,
+  useGetSourceUploadInfoQuery,
+} from '../../../../store/provisioningApi';
+import { useShowActivationKeyQuery } from '../../../../store/rhsmApi';
+import {
+  selectActivationKey,
+  selectArchitecture,
+  selectAwsAccountId,
+  selectAwsShareMethod,
+  selectAwsSource,
+  selectBlueprintDescription,
+  selectBlueprintName,
+  selectDistribution,
+} from '../../../../store/wizardSlice';
+import { useGetEnvironment } from '../../../../Utilities/useGetEnvironment';
+
+const ExpirationWarning = () => {
+  return (
+    <div className="pf-u-mr-sm pf-u-font-size-sm pf-u-warning-color-100">
+      <ExclamationTriangleIcon /> Expires 14 days after creation
+    </div>
+  );
+};
+
+export const ImageOutputList = () => {
+  const distribution = useAppSelector((state) => selectDistribution(state));
+  const arch = useAppSelector((state) => selectArchitecture(state));
+  return (
+    <TextContent>
+      <ReleaseLifecycle />
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Release
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {RELEASES.get(distribution)}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Architecture
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>{arch}</TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+export const TargetEnvAWSList = () => {
+  const { data: rawAWSSources, isSuccess } = useGetSourceListQuery({
+    provider: 'aws',
+  });
+
+  const source = useAppSelector((state) => selectAwsSource(state));
+  const awsAccountId = useAppSelector((state) => selectAwsAccountId(state));
+  const awsShareMethod = useAppSelector((state) => selectAwsShareMethod(state));
+
+  const { data, isError } = useGetSourceUploadInfoQuery(
+    {
+      id: parseInt(source?.id as string),
+    },
+    { skip: source === undefined }
+  );
```

Comment:
I can't see the data or isError being used anywhere in the component. Are those needed?